modules/delegates.js

```diff
@@ -353,112 +353,160 @@ __private.decryptPassphrase = function(encryptedPassphrase, password) {
  * @returns {setImmediateCallback} cb, err
  * @todo Add description for the params
  */
-__private.checkDelegates = function(publicKey, votes, state, cb, tx) {
```

Comment:
`publicKey` is just identifier of account for which you want to check the votes. Renaming it to `senderPublicKey` make it confusing.

Reply:
Every account has pubicKey, and previously we have variable shadowing for `publicKey`. It makes sense to have a more descriptive name. So we avoid making same mistakes again.